packages/create-pantheon-decoupled-kit/src/templates/next-wp/.env.example

```diff
@@ -1,7 +1,21 @@
-# Optional environment variables are commented out
-WPGRAPHQL_URL=
-IMAGE_DOMAIN=
-#PANTHEON_UPLOAD_PATH=/example
-#PREVIEW_SECRET=
-#WP_APPLICATION_USERNAME=
-#WP_APPLICATION_PASSWORD=
+# Copy as .env.development.local to override envars for local development
+
+# The graphql endpoint of your WordPress instance
+WPGRAPHQL_URL=https://my-backend-site.pantheonsite.io/wp/graphql
+
+# The secret set on your WordPress Preview Site. Required for preview functionality
```

Comment:
How/Where do I (the user/developer) find out what this secret is? Can we add a link to where I would find it?

Reply:
Also, please explain "preview" more. Maybe something like


> This variable is necessary when content editors use preview functionality in the WordPress interface to see/preview a content change before that content change becomes public. This front-end application uses this secret to retrieve the not-yet-public content from WordPress. To find this value....

Reply:
> Also, please explain "preview" more. Maybe something like ...

Would a link to our docs suffice?
https://decoupledkit.pantheon.io/docs/frontend-starters/nextjs/nextjs-drupal/implementing-preview

Reply:
Personally I prefer a link to documentation to a paragraph of explanation. Seems to verbose for this file.

If we add that, we'd need to use the Drupal specific link here.